relay-server/src/actors/spooler/mod.rs

```diff
@@ -209,8 +211,12 @@ impl InMemory {
             count += current_count;
             self.used_memory -= current_size;
         }
+        self.envelope_count -= count as isize;
         relay_statsd::metric!(
-            histogram(RelayHistograms::BufferEnvelopesMemory) = self.used_memory as f64
+            histogram(RelayHistograms::BufferEnvelopesMemoryBytes) = self.used_memory as f64
+        );
+        relay_statsd::metric!(
+            histogram(RelayHistograms::BufferEnvelopesMemoryCount) = self.envelope_count as u64
```

Comment:
Why not make it a counter instead? Is it because of potential negative counts?

Reply:
This logs the number of envelopes in the buffer at a certain time. A counter is strictly incremental, so depending on how often you log the counter, you get different values. It's usually used for counting rather than for tracking the number of things.

@jjbayer I think you actually want to log a gauge here.

Reply:
> I think you actually want to log a gauge here.

You're right. Shouldn't `EnvelopeQueueSize` also be a gauge then?